components/monitoring/logging/staging/kustomization.yaml

```diff
@@ -12,9 +12,21 @@ patches:
       group: external-secrets.io
       version: v1beta1
       kind: ExternalSecret
-  - path: configure-logging/cluster-log-forwarder-patch.yaml
-    target:
-      group: logging.openshift.io
-      version: v1
-      kind: ClusterLogForwarder
-      name: instance
+  - target:
+      group: external-secrets.io
+      version: v1beta1
+      kind: ExternalSecret
+      name: log-forwarder-splunk-rhtap-application-external-secret
+    patch: |
+      - op: replace
+        path: /spec/dataFrom/0/extract/key
+        value: production/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-application
+  - target:
+      group: external-secrets.io
+      version: v1beta1
+      kind: ExternalSecret
+      name: log-forwarder-splunk-rhtap-audit-external-secret
+    patch: |
```

Comment:
can you explain how that works?

Reply:
Are you asking about Kustomize's patching mechanism? 
Can you be more specific?

Reply:
Yes, I'm not familiar with the syntax

Reply:
@Omeramsc It patches configuration which is in the base folder. 
If you wish, we can have a short meeting and I'll explain short the mechanism works.